docs/providers/kubeless/guide/installation.md

```diff
@@ -0,0 +1,109 @@
+<!--
+title: Serverless Framework - Kubeless Guide - Installing The Serverless Framework and Kubeless
+menuText: Installation
+menuOrder: 3
+description: How to install the Serverless Framework and start using it with Kubeless
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/providers/kubeless/guide/installation)
+<!-- DOCS-SITE-LINK:END -->
+
+# Kubeless - Installation
+
+## Installing Kubeless in your Kubernetes cluster
+
+Kubeless runs on [Kubernetes](https://kubernetes.io), you need a working Kubernetes cluster to run kubeless. For testing you can use [minikube](https://github.com/kubernetes/minikube).
+
+You should deploy Kubeless in your cluster using one of YAML manifests found in the release package. It will create a *kubeless* Namespace and a *function* ThirdPartyResource. You will see a *kubeless* Controller, and *kafka*, *zookeeper* StatefulSet running.
+
+There are several kubeless manifests being shipped for multiple k8s environments (non-rbac and rbac), please consider to pick up the correct one:
+
+* *kubeless-$RELEASE.yaml* is used for non-RBAC Kubernetes cluster.
+* *kubeless-rbac-$RELEASE.yaml* is used for RBAC-enabled Kubernetes cluster.
+
+For example, this below is a show case of deploying kubeless to a non-RBAC Kubernetes cluster.
+
+	$ export RELEASE=0.0.20
```

Comment:
I would point here to the kubeless README if possible, it is going to be easier to maintain.